plugins/hls-alternate-number-format-plugin/hls-alternate-number-format-plugin.cabal

```diff
@@ -0,0 +1,61 @@
+cabal-version:      2.4
+name:               hls-alternate-number-format-plugin
+version:            0.1.0.0
+synopsis:           Provide Alternate Number Formats plugin for Haskell Language Server
+description:
+  Please see the README on GitHub at <https://github.com/haskell/haskell-language-server#readme>
+
+license:            Apache-2.0
+license-file:       LICENSE
+author:             Nick Suchecki
+maintainer:         nicksuchecki@gmail.com
+category:           Development
+build-type:         Simple
+extra-source-files: LICENSE
+
+library
+  exposed-modules:  Ide.Plugin.AlternateNumberFormat
+  other-modules:    Ide.Plugin.Literals, Ide.Plugin.Conversion
```

Comment:
You use `Ide.Plugin.Conversion` in your tests, so it needs to be exposed.

Reply:
Yep, not sure how I missed that originally.